hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -90,8 +85,7 @@
    */
   @SuppressWarnings("unchecked")
```

Comment:
Only if you need to update the patch anyway: this `SuppressWarnings` is unnecessary, can be removed.